aria-practices.html

```diff
@@ -3734,6 +3734,390 @@ <h5>Examples</h5>
+          The <code>timer</code> role is a subclass of the <code>status</code> role.
+          The default value of <code>aria-live</code> value of <code>off</code>.
+        </p>
+
+        <p>
+          The <code>marquee</code> role indicates non-essential information that changes frequently.
+          The default value of <code>aria-live</code> value of <code>off</code>.
+        </p>
+
+        <p class="note">
+          <a href="https://github.com/w3c/aria/issues/1104">Issue 1104 for the ARIA specification</a> proposes deprecating these roles.
+        </p>
+      </section>
+    </section>
+  </section>
+
+  <section id="defer_exposing_content_updates_using_aria-busy">
+    <h2>Defer Exposing Content Updates using <code>aria-busy</code></h2>
+
+    <p>
+      Sometimes, it takes some time for a script to update the content of a live region or a widget.
+      For example, it could be waiting on a network response.
+      In order to avoid rendering half-baked content to users of assistive technology,
+      the <code>aria-busy</code> attribute can be set to <code>true</code> while the content is being updated,
+      and then to <code>false</code> when it is done.
+    </p>
+
+    <p>
+      The ARIA specification allows, but does not require, assistive technologies to wait until <code>aria-busy</code> is changed to <code>false</code> before exposing content changes to the user.
+      So assistive technologies are allowed to ignore the <code>aria-busy</code> attribute.
+    </p>
+
+    <p>
+      For assistive technologies that honor the <code>aria-busy</code> attribute, one possible implementation strategy is to not announce any content for elements when <code>aria-busy</code> is set to <code>true</code>.
+      Therefore, it is important that the attribute is changed back to <code>false</code> when the content ought to be available to the user.
+      For example, when the update is complete, but also when there is an error, or if the update is aborted.
+      Otherwise, content could be left inaccessible to assistive technology users.
+    </p>
+
+    <p>
+      Consider the search form from the earlier example in the <a href="#live-regions">Live Resions</a> section.
+      When the user starts typing a new search into the search field, the script would update the search results live region,
+      and maybe update multiple times as new search results appear,
+      but it would be a better experience for users of assistive technology to only be notified when the new search is complete.
+    </p>
+
+    <pre><code>&lt;form role="search" aria-labelledby="search">
+ &lt;h2 id="search">Search&lt;/h2>
+ &lt;label>Search query: &lt;input type="search" name="q" oninput="updateSearch(event)">&lt;/label>
+ &lt;div id="search-result-status" role="region" aria-live="polite">&lt;/div>
```

Comment:
See earlier discussion about `role="region"`.
For this example, consider simply using `role="status"` (in which case, `aria-live="polite"` can be removed).